rfd/0081/health-check-draft.md

```diff
@@ -0,0 +1,58 @@
+# Mariposa Health Check Service
+
+The Mariposa (RFD36) health check service will be responsible for monitoring the health of user services. It establishes a set of supported health checks and provides an API for defining health checks that should be ran against project services. These health check definitions will be persisted in a proper data store (Postgres/Manatee?).
```

Comment:
> and it's connected to the private networks of the user

That point is significant because there's nothing in Triton DataCenter that can route a request to a user's private fabrics without having a NIC on that fabric. The NAT zone does have that NIC, so...